app/src/main/java/io/github/androidacademyglobal/AppActivity.kt

```diff
@@ -22,22 +22,22 @@ class AppActivity : ComponentActivity() {
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
                 ) {
-                    Greeting(io.github.anroid_acagemy_global.core.Greeting().greeting())
+                    Greeting("Android")
                 }
             }
         }
     }
-}
 
-@Composable
-fun Greeting(text: String) {
-    Text(text = text)
-}
+    @Composable
+    fun Greeting(name: String) {
+        Text(text = "Hello $name!")
+    }
 
-@Preview(showBackground = true)
-@Composable
-fun DefaultPreview() {
-    AndroidAcademyAppTheme {
-        Greeting("Android")
+    @Preview(showBackground = true)
+    @Composable
+    fun DefaultPreview() {
```

Comment:
private fun DefaultPreview() = AndroidAcademyAppTheme {...}
and out of class if stateless

Reply:
deleted this